mscore/palette.cpp

```diff
@@ -276,7 +282,7 @@ static void applyDrop(Score* score, ScoreView* viewer, Element* target, Element*
       DropData dropData;
       dropData.view       = viewer;
       dropData.pos        = pt;
-      dropData.dragOffset = pt;
+      dropData.dragOffset = QPointF();
```

Comment:
Further investigation led me to discover that target->pagePos() is actually good here.  I guess it needs to be score-relative only for measures, page-relative for everything else?  So I have updated this.